users/api/http/transport.go

```diff
@@ -56,6 +56,13 @@ func MakeHandler(svc users.Service, tracer opentracing.Tracer, l log.Logger) htt
 		opts...,
 	))
 
+	mux.Put("/users", kithttp.NewServer(
```

Comment:
Can we move this near to GET on `/users` (or vice-versa) - to increase readability.

Reply:
Indeed, sorry for this remark :)